UOP1_Project/Assets/Scripts/EditorTools/SwapPrefabs/Editor/SwapPrefabs.cs

```diff
@@ -0,0 +1,584 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+namespace ChopChop.EditorTools.jandd661
```

Comment:
We've been using the format of UOP1.[SystemName]. The conventions document should probably be updated to include this.

Reply:
Yes it should.